after/ftplugin/xml.vim

```diff
@@ -1,12 +1,6 @@
 if gradle#isGradleProject()
-
-  call gradle#setClassPath()
-  call gradle#setupGradleCommands()
-
   if android#isAndroidProject()
-    call android#setAndroidSdkTags()
-    call android#setClassPath()
-    call android#setupAndroidCommands()
+    autocmd BufEnter *.xml XMLns android
```

Comment:
No need to have autocmd here as this is already an ftplugin script. The following should suffice:

    XMLns android